ui/src/services/hooks/health.ts

```diff
@@ -23,7 +23,7 @@ export const getDockerStatus = async (): Promise<Health> => {
 export const useLocalStackHealth = (): UseResourcesGraph => {
   const { data, error, isValidating, mutate } = useSWR<Health>(
     SwrCacheKey.HEALTH,
-    () => getDockerStatus(), { revalidateOnMount: true, refreshInterval: 2000 },
+    () => getDockerStatus(), { revalidateOnMount: true },
```

Comment:
It will update just not constantly......If LocalStack is not running you get an on the console an error every 2 seconds which is quite annoying...There is still the button to refresh manually in case .... maybe I can re-fetch each time the component re-renders?

Reply:
I think in this case we could ignore what's happening on the console, as users are not able to see that anyway (correct me if I am wrong). 

One more thought I had regarding the navbar was to switch these two components:
![image](https://user-images.githubusercontent.com/39307517/208079240-7e26e642-b6fd-4727-b695-2f9a926b4362.png)

This would allow to keep the layout consistent when starting LS with the start button.